platform/pricemodel/src/pricemodel/entrypoint.py

```diff
@@ -94,9 +79,9 @@ def invocations() -> Predictions | None:
 
     equity_bars_raw_data_grouped_by_ticker = equity_bars_raw_data.groupby("ticker")
 
-    predictions = {}
+    predictions: dict[str, list[float]] = {}
     for ticker, ticker_bars_raw_data in equity_bars_raw_data_grouped_by_ticker:
-        ticker_predictions = price_model.get_predictions(
+        ticker_predictions: list[float] = price_model.get_predictions(
             data=ticker_bars_raw_data,
         )
```

Comment:
It'll probably be a part of one of my "data" issues but we'll also want to write the prediction payload to the data layer. Also, a CloudEvent should be emitted here.

Reply:
cloud events coming in the upstream PR